src/components/AnonymousChat/child/ChannelTitle.tsx

```diff
@@ -112,9 +112,7 @@ const ChannelTitle = ({
   }
 
   const getTitle = () => {
-    const isMyPost = type?.includes('MY_SIGNED_POST') || type?.includes('MY_ANON_POST');
-
-    if (isMyPost) {
+    if (isMe) {
       if (message?.length === 0) return 'Your media post 📸 🖼';
       return 'Your post: ';
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `isMyPost` to `isMe` seems to be a simplification of the logic. However, it's important to ensure that `isMe` is correctly set in all contexts where this component is used. If `isMe` only checks if the current user is the one who created the post, it might not cover all cases previously covered by `isMyPost`. Please verify that `isMe` is equivalent to the previous condition (`type?.includes('MY_SIGNED_POST') || type?.includes('MY_ANON_POST')`) in all usage scenarios.
```diff
-    if (isMe) {
+    const isMyPost = type?.includes('MY_SIGNED_POST') || type?.includes('MY_ANON_POST');
+    if (isMyPost) {
       if (message?.length === 0) return 'Your media post 📸 🖼';
       return 'Your post: ';
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->